src/www/vpn_openvpn_server.php

```diff
@@ -1508,19 +1508,19 @@ function tuntap_change() {
                         <input name="dns_server_enable" type="checkbox" id="dns_server_enable" value="yes" <?=!empty($pconfig['dns_server1']) || !empty($pconfig['dns_server2']) || !empty($pconfig['dns_server3']) || !empty($pconfig['dns_server4']) ? "checked=\"checked\"" : "" ;?> onclick="dns_server_change()" />
                         <div id="dns_server_data" summary="dns servers">
                               <span>
-                                <?=gettext("Server"); ?> #1:&nbsp;
+                                <?=gettext("Server #1:"); ?>&nbsp;
```

Comment:
In the future these things should be set programatically: sprintf(gettext('Server #%s'), 1); etc. (will merge now anyway, it's only 4 strings).